tracer/test/Datadog.Trace.Tools.Runner.IntegrationTests/Checks/IisCheckTests.cs

```diff
@@ -0,0 +1,145 @@
+// <copyright file="IisCheckTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NETCOREAPP3_1
+
+using System.Diagnostics;
+using System.IO;
+using System.Net.Http;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Datadog.Trace.TestHelpers;
+using Datadog.Trace.Tools.Runner.Checks;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.Tools.Runner.IntegrationTests.Checks
+{
+    [Collection(nameof(ConsoleTestsCollection))]
+    public class IisCheckTests : TestHelper, IClassFixture<IisFixture>
+    {
+        private readonly IisFixture _iisFixture;
+
+        public IisCheckTests(IisFixture iisFixture, ITestOutputHelper output)
+            : base("AspNetCoreMvc31", output)
+        {
+            _iisFixture = iisFixture;
+            _iisFixture.ShutdownPath = "/shutdown";
+        }
+
+        [SkippableTheory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public async Task WorkingApp(bool mixedRuntimes)
+        {
+            EnsureWindowsAndX64();
+
+            var siteName = mixedRuntimes ? "sample/mixed" : "sample";
+
+            var buildPs1 = Path.Combine(EnvironmentTools.GetSolutionDirectory(), "tracer", "build.ps1");
+
+            try
+            {
+                // GacFixture is not compatible with .NET Core, use the Nuke target instead
+                Process.Start("powershell", $"{buildPs1} GacAdd --framework net461").WaitForExit();
```

Comment:
😮 Can't believe you got this to work

Reply:
I wouldn't be surprised if that's the reason why it takes minutes to start